setup.py

```diff
@@ -86,7 +87,7 @@
     ],
     keywords="water systems, chemical engineering, process modeling, filtration, desalination, nawi",
     packages=find_namespace_packages(),
-    python_requires=">=3.7, <4",
+    python_requires=">=3.7,<3.11",
```

Comment:
```suggestion
    python_requires=">=3.7",
```
Is the `<3.11` worth it? Before this PR WaterTAP would happily install on 3.10...